hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/Processor.java

```diff
@@ -254,12 +247,6 @@ default boolean finishSnapshotRestore() {
          */
         int globalProcessorIndex();
```

Comment:
We should document here that it's zero-based and that it's in the range 0 .. totalParallelism-1